mono/mini/mini-x86.c

```diff
@@ -5425,7 +5425,8 @@ mono_arch_emit_epilog (MonoCompile *cfg)
 	if (need_stack_frame)
 		x86_leave (code);
 
-	if (CALLCONV_IS_STDCALL (sig)) {
+	if (CALLCONV_IS_STDCALL (sig) &&
+		method->wrapper_type != MONO_WRAPPER_MANAGED_TO_NATIVE) {
 		MonoJitArgumentInfo *arg_info = alloca (sizeof (MonoJitArgumentInfo) * (sig->param_count + 1));
```

Comment:
Why is this needed ? The managed-to-native wrappers are normal managed methods, they shouldn't have an stdcall calling convention.